cli/cmd/flags.go

```diff
@@ -64,19 +66,37 @@ func bindRPCURL(cmd *cobra.Command, rpcURL *string) {
 func bindUnjailConfig(cmd *cobra.Command, cfg *unjailConfig) {
 	netconf.BindFlag(cmd.Flags(), &cfg.Network)
 	bindPrivateKeyFile(cmd, &cfg.PrivateKeyFile)
-	_ = cmd.MarkFlagRequired("network")
+	_ = cmd.MarkFlagRequired(flagNetwork)
+}
+
+func bindEOAConfig(cmd *cobra.Command, cfg *eoaConfig) {
+	bindPrivateKeyFile(cmd, &cfg.privateKeyFile)
+	netconf.BindFlag(cmd.Flags(), &cfg.network)
+
+	cmd.Flags().StringVar(&cfg.evmRPC, "evm-rpc", "", "Optional Omni EVM execution RPC API endpoint. Defaults to consensus.<network>.omni.network")
```

Comment:
lets rename this to `execution-rpc`, and lets add `consensus-rpc`. Users might want to specify their own halo node.